cmd/frontend/internal/httpapi/graphql.go

```diff
@@ -125,6 +127,9 @@ func serveGraphQL(logger log.Logger, schema *graphql.Schema, rlw graphqlbackend.
 			}
 
 			if !isInternal && (cost.FieldCount > graphqlbackend.MaxFieldCount) {
+				if envvar.SourcegraphDotComMode() { // temporarily logging queries that exceed field count limit on Sourcegraph.com
+					logger.Warn(fmt.Sprintf("GQL cost limit exceeded. Query: (%s), Vars: (%s)", params.Query, params.Variables))
```

Comment:
The idiomatic way of logging parameters is to use this API, then we can also do proper JSON magic on the field values in GCP. 

```suggestion
					logger.Warn("GQL cost limit exceeded", log.String("query", params.Query))
```

In the suggestion I left out variables because it could contain secrets. I'm not sure if that's a problem.